Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -238,7 +237,6 @@ func LoadFromFile(filename string) (*clientcmdapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	glog.V(3).Infoln("Config loaded from file", filename)
```

Comment:
> @deads2k @liggitt the extended tests logs are flooded by these messaging. It only gives you what kubeconfig is being used. In case of mysql or other extended tests, this will print out >100 messages to log ;-)

If you were only disabling it for extended tests, I'd still be unhappy, but not super unhappy.  This is also removing it so that I can't tell someone to set `oc get --loglevel=3` and see what's actually being used.